packages/merge-styles/src/IStyleSet.ts

```diff
@@ -22,7 +22,7 @@ export type __MapToFunctionType<T> = Extract<T, Function> extends never
  * It may optionally contain style functions for sub components in the special `subComponentStyles`
  * property.
  */
-export type IStyleSet<TStyleSet extends IStyleSet<TStyleSet>> = {
+export type IStyleSet<TStyleSet extends IStyleSet<TStyleSet> = { [key: string]: any }> = {
```

Comment:
why is this change needed?

Reply:
Originally when defining all of the concat/merge style definitions, I had provided the default in all of the input use cases as`IStyleSet<any>`. That felt wrong for a number of reasons, and instead I moved the default initializer here (to control the default in one place.) Then I thought `any` was too liberal and scoped it to default to an object.